gtest-parallel

```diff
@@ -322,8 +353,6 @@ parser.add_option('--gtest_filter', type='string', default='',
                   help='test filter')
 parser.add_option('--gtest_also_run_disabled_tests', action='store_true',
                   default=False, help='run disabled tests too')
-parser.add_option('--format', type='string', default='filter',
-                  help='output format (raw,filter)')
```

Comment:
Removing this flag is a behavior change. Why is it in this patch, and are there any other behavior changes?